compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -262,7 +266,7 @@ object Types {
       }
 
     /** Is this type exactly Nothing (no vars, aliases, refinements etc allowed)? */
-    def isBottomType(using Context): Boolean = this match {
+    def isNothingType(using Context): Boolean = this match {
```

Comment:
Nitpick: I would call it `IsNothing` instead

Reply:
oops, lowercase `isNothing` I mean

Reply:
Renamed